tests/bluetooth/audio/ascs/mocks/ascs_help_utils.c

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2023 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/bluetooth/gatt.h>
+
+#include "ascs_help_utils.h"
+
+const struct bt_gatt_attr *ascs_get_ase_control_point_attr(void)
+{
+	const struct bt_gatt_service_static *service = ascs_get_gatt_service();
+
+	for (size_t i = 0; i < service->attr_count; i++) {
+		if (!bt_uuid_cmp(service->attrs[i].uuid, BT_UUID_ASCS_ASE_CP)) {
+			return &service->attrs[i];
+		}
+	}
+
+	return NULL;
+}
+
+void ascs_foreach_ase_attr(ascs_foreach_ase_attr_func_t func, void *user_data)
+{
+	const struct bt_gatt_service_static *service = ascs_get_gatt_service();
+	uint8_t result;
+
+	for (size_t i = 0; i < service->attr_count; i++) {
+		const struct bt_gatt_attr *attr = &service->attrs[i];
+
+		if (bt_uuid_cmp(attr->uuid, BT_UUID_ASCS_ASE_SNK) ||
+		    bt_uuid_cmp(attr->uuid, BT_UUID_ASCS_ASE_SRC)) {
+			continue;
+		}
+
+		result = func(attr, user_data);
+		if (result == BT_GATT_ITER_STOP) {
+			return;
+		}
+	}
+}
```

Comment:
I think this is a duplicate of `bt_audio_unicast_server_foreach_ep`, isn't it?

Reply:
Not really, as the bt_audio_unicast_server_foreach_ep iterates non-idle endpoints only